agents/craftassist/craftassist_agent.py

```diff
@@ -219,10 +220,12 @@ def init_perception(self):
             low_level_data=self.low_level_data,
             mark_airtouching_blocks=self.mark_airtouching_blocks,
         )
-        # set up the SubComponentClassifier model
-        if os.path.isfile(self.opts.semseg_model_path):
-            self.perception_modules["semseg"] = SubcomponentClassifierWrapper(
-                self, self.opts.semseg_model_path, low_level_data=self.low_level_data
+        # set up the detection model
+        # TODO: @kavya to check that this gets passed in when running the agent
+        # TODO: fetch text_span here ?
+        if self.opts.detection_model_path and os.path.isfile(self.opts.detection_model_path):
```

Comment:
don't do this here, am handling it in perceive()

Reply:
Sounds good, I can remove this, had put this as a placeholder